scripts/core/queue.yml

```diff
@@ -62,6 +62,12 @@ etors:
     - name: PRIORITY_HIGH
       value: "$X_BIT(6)"
       desc: "High priority queue"
+    - name: SUBMISSION_BATCHED
+      value: "$X_BIT(7)"
+      desc: "Enqueue and submit in a batch later, as determined by adaptor or driver"
+    - name: SUBMISSION_IMMEDIATE
+      value: "$X_BIT(8)"
+      desc: "Enqueue and submit immediately"
```

Comment:
I think these could do with a remark in the descriptions on whether they are performance hints with no semantic impact on API usage, or otherwise if they're expected to affect/be affected by stuff like the `QueueFlush` entry point

Reply:
Changed "desc" field to say these properties are hints to the implementation and do not affect queue semantics.

Reply:
thanks, you'll need to run the generate target again so the changes are reflected in the headers etc.

Reply:
Ok, done.